.github/workflows/terraform.yml

```diff
@@ -35,7 +35,7 @@ jobs:
         env:
           BASE_SHA: ${{ github.event.pull_request.base.sha }}
         run: |
-          JQ_OUTPUT_CHANGED=$(git diff --name-only $BASE_SHA $GITHUB_SHA modules/*/*/*.tf |
+          JQ_OUTPUT_CHANGED=$(git diff --name-only $BASE_SHA $GITHUB_SHA modules/**/*.tf |
```

Comment:
I don't think this should be changed - the example folders aren't modules in their own right

Reply:
So I will argue that these are now tests, and justify the running of the Terraform Modules.